### ISSUE TITLE
404 on a lot of links!

### ISSUE THREAD

**Issue by Grogdunn** [mojohaus/mojohaus.github.io#10]
A lot of links are broken in this page http://www.mojohaus.org/plugins.html
eg:
- jaxb2
- xmlbeans
- sqlj
- etc...

**Comment by sheix** [comment]
Also was6! I'm unable to find it elsewhere on the web, and I'm looking forward to use it.

**Comment by GurvanLd** [comment]
Also : New location for javascript tools : https://github.com/softec/javascript-maven-tools

**Comment by mattbrunetti** [comment]
Thanks @GurvanLd for that link, as I was looking for some plugin(s) for javascript

There seems to be more broken links than working ones:
http://www.mojohaus.org/l10n-maven-plugin/
http://www.mojohaus.org/javascript-maven-tools/
http://www.mojohaus.org/jslint-maven-plugin/
http://www.mojohaus.org/js-import-maven-plugin/
...

**Comment by andham** [comment]
Yes, lots of the links are broken. Also see #6 as some plugins probably wasn't moved to Github when Codehaus was shut down.
PRs for fixing this this are welcome!

**Comment by andham** [comment]
I've fixed the sqlj-maven-plugin link by publishing a site of the 1.4-SNAPSHOT version.

**Comment by mbmast** [comment]
Another broken link https://www.mojohaus.org/sablecc-maven-plugin/

**Comment by dmatej** [comment]
It is more simple to list plugins which do have their page. I'm trying to find out how to create a page for native2ascii-maven-plugin, do we have some documentation for that?

**Comment by slachiewicz** [comment]
> It is more simple to list plugins which do have their page. I'm trying to find out how to create a page for native2ascii-maven-plugin, do we have some documentation for that?

please check in gh repo settings tab, then Pages. For versions repo we have this:
<img width="648" alt="image" src="https://github.com/mojohaus/mojohaus.github.io/assets/6705942/46f6f548-b5aa-4e5c-974a-4ea8460aefb3">

**Comment by dmatej** [comment]
> please check in gh repo settings tab, then Pages. 

Unfortunatelly I don't have permissions to visit the repo settings. I tried to visit other plugins and then executed 

```
mvn verify site -DperformRelease scm-publish:publish-scm
```
probably just few minutes after you, so I removed my changes now. Sources are in place, but I cannot view and change these settings. I can see just this menu:
![image](https://github.com/mojohaus/mojohaus.github.io/assets/302904/60a0c09d-2f5f-4ab7-8b84-58a8cf87e6e8)
